scipy/fftpack/basic.py

```diff
@@ -215,8 +215,6 @@ def fft(x, n=None, axis=-1, overwrite_x=False):
 
             y(j) = sum[k=0..n-1] x[k] * exp(-sqrt(-1)*j*k* 2*pi/n), j = 0..n-1
 
-        Note that ``y(-j) = y(n-j).conjugate()``.
```

Comment:
> But this is true only if the input x is a real-valued function, yet the statement appears even under complex fft. Better to omit this.

I suggest to clarify rather than delete, i.e. fix the indexing and just add `for real-valued x`.